docs/performance/storage.md

```diff
@@ -204,11 +230,36 @@ CREATE TABLE IF NOT EXISTS "doc"."taxi_nocolumnstore"(
 
 ### Effects on storage
 
+CrateDB v5.9.3
+
 | avg_mib_per_shard | total_mib |
 |-------------------|-----------|
 | 53                | 639       |
 
-Data was reduced: `~13.11%`, similar to `no_index`.
+CrateDB v5.10.2
+
+| avg_mib_per_shard | total_mib |
+|-------------------|-----------|
+| 39                | 471       |
+
+Data was reduced `~13.11%` in v5.9 similar to `no_index`.
+Data was increased `~10%` in v.5.10
+
+> NOTE: In v5.10, we don’t recommend that you turn off column storage.
+
+## Upgrade to v5.10 and Optimize table
+
+In v5.10 we introduced a new table storage format that can reduce storage up to 50% in comparison
+to previous versions.
+
+The easiest way to gain massive storage reduction is to update to the latest version available. 
```

Comment:
I would suggest to remove the additional line break here.
```suggestion
In v5.10 we introduced a new table storage format that can reduce storage up to 50% in comparison
to previous versions. The easiest way to gain massive storage reduction is to update to the latest version available.
```

Reply:
You were right! It didn't change storage usage until I merged segments.